ksqldb-engine/src/main/java/io/confluent/ksql/function/InternalFunctionRegistry.java

```diff
@@ -106,10 +106,7 @@ public boolean isPresent(final FunctionName functionName) {
     if (udafs.containsKey(functionName.text().toUpperCase())) {
       return true;
     }
-    if (udtfs.containsKey(functionName.text().toUpperCase())) {
-      return true;
-    }
-    return false;
+    return udtfs.containsKey(functionName.text().toUpperCase());
```

Comment:
nice clean up

Reply:
Thanks!  To be honest, I found these via IntelliJ inspections while I was working on other UDF/UDAF work.:)